README.md

```diff
@@ -9,3 +10,8 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+Proof
+
+- We have two graphs $A=(V_1 , E_1)$ and $B=(V_2 , E_2)$ that are isomorphic to eachother and to themselves (i.e. they are both one to one and onto). There exists a disconnected node in $A$ therefore $A$ is not completley connected. If there is a disconnected node in $A$ such that $(u,v) \notin E_1$ and $A$ and $B$ have a one to one relation $f: V_1 \rightarrow V_2$ such that every node in $A$ is mapped to one element in $B$ and $A$ and $B$ have a onto relation $f: V_1 \rightarrow V_2$ such that each node in $A$ is mapped to a corresponding element of $B$, then there must also be disconnected node in $B$ such that  $(f(u),f(v)) \notin E_2$. Since $A$ and $B$ are isomorphic and there is a one to one and onto mapping even when $A$ and $B$ are not completley connected graphs, then if two graphs $A$ and $B$ are isomorphic they do not have to be completely connected.
```

Comment:
Still not there. Your argument is about what happens *if* two graphs are isomorphic and *if* there is an edge missing. You need to show that this can be true.